### DISCUSSION TITLE
Aug 19, 2021 full team meeting minutes

### DISCUSSION THREAD

**Discussion by mli1** [SafetyGraphics/safetyGraphics#624] [Meetings]
Attendees:
@JimBuchanan @mli1 @zskrivan @PHayashi @anzianorj @jwildfire @DOBrien1 @Scott Wong @spduke @wal615 @Rachel Dlugash @hendricksonb @Ranjeeta Sinvhal @MelvinSMunsaka @Michael Colopy

Meeting notes:

1- Beta testing. Zak volunteered to propose user acceptancy criteria for beta testing. Jeremy to follow up with Zak on this.

2- PHUSE and ISG(ASA) collaboration: Both are under PPP with FDA. Strategy should be considered for future collaboration. 

> AE volcano plot might be a topic for collaboration. ISG group is developing an interactive AE volcano plot, Zak and his team has been using an AE volcano plot that he developed for interactive submission to FDA and EMA. Zak is willing to share with ISG team on the feedback from both agencies. **_Post meeting note:_** Dennis has reached out to Zak for offline meetings.

3- Subgroup update: 

> QT: team has developed a mockup of what the visualization may look like. Aiming for prototype programming in late Sep/early Oct.

_**Team, Please kindly help to update the meeting minutes as I was not able to fully dedicated to this meeting. Thank you!! Mengchun--- to be deleted when this is finalized**_

**Comment by MelvinSMunsaka** [comment]
Zak - happy to help out with Beta testing. Are you thinking of what we did the first time around? 

Melvin
